### PR TITLE
[zh-cn] update `::after`

### PR DESCRIPTION
**Pull request by jasonren0403**

### CODE REVIEW

**Comment by Kimhyuntae9665** [comment]
hey bro , how did you fix this Lint and review content files / lint-and-review-docs (pull_request_target)  error ?

Logs from markdownlint:
yarn run v1.22.22
$ /home/runner/work/translated-content/translated-content/node_modules/.bin/markdownlint-cli2 --fix files/zh-cn/web/css/_doublecolon_after/index.md
markdownlint-cli2 v0.13.0 (markdownlint v0.34.0)
Finding: files/zh-cn/web/css/_doublecolon_after/index.md !node_modules !.git !.github !**/conflicting/** !**/orphaned/**
Linting: 1 file(s)
Summary: 0 error(s)
Done in 0.30s.

Logs from front-matter linter:
yarn run v1.22.22
$ node scripts/front-matter_linter.js --fix true --config-file /home/runner/work/translated-content/translated-content/front-matter-config.json /home/runner/work/translated-content/translated-content/files/zh-cn/web/css/_doublecolon_after/index.md
0 0
Done in 0.55s.

Please fix all the linting issues mentioned in above logs and in the review comments.
Error: Process completed with exit code 1.

**Comment by jasonren0403** [comment]
> hey bro , how did you fix this Lint and review content files / lint-and-review-docs (pull_request_target) error ?

We auto detects lint error on each commit, and the auto-linter bot can point them out for us to fix.

File: files/zh-cn/web/css/_doublecolon_after/index.md [hunk 1]
```diff
@@ -1,38 +1,47 @@
...
 ---
 
 {{CSSRef}}
 
-CSS[伪元素](/zh-CN/CSS/Pseudo-elements)`::after`用来创建一个伪元素，作为已选中元素的最后一个子元素。通常会配合{{ cssxref("content") }}属性来为该元素添加装饰内容。这个虚拟元素默认是行内元素。
+在 CSS 中，**`::after`** 会创建一个[伪元素](/zh-CN/docs/Web/CSS/Pseudo-elements)，作为所选元素的最后一个子元素。它通常用于为具有 {{CSSxRef("content")}} 属性的元素添加修饰内容。默认情况下，它是行向布局的。
 
 {{EmbedInteractiveExample("pages/tabbed/pseudo-element-after.html", "tabbed-standard")}}
 
+> **备注：** `::before` 和 `::after` 生成的伪元素是行级盒子，就好像它们是应用它们的元素或“源元素”的直接子元素，因此不能应用于[_替换元素_](/zh-CN/docs/Web/CSS/Replaced_element)（如 {{htmlelement("img")}}），它们的内容被替换，不受当前文档样式的影响。
```
Comment (yin1999) [inline files/zh-cn/web/css/_doublecolon_after/index.md:14]: How abount this?

```suggestion
> **备注：** `::before` 和 `::after` 生成的伪元素是行级盒子，就好像它们是应用它们的元素或“源元素”的直接子元素，因此不能应用于[_替换元素_](/zh-CN/docs/Web/CSS/Replaced_element)（如 {{htmlelement("img")}}），它们的内容在不受当前文档样式的影响的情况下被替换。
```

**Review by yin1999 — APPROVED** [review]
LGTM. Just one consideration

**Pull request merged**